.github/workflows/release.yml

```diff
@@ -38,19 +38,11 @@ jobs:
 
       - name: Check Release Version
         id: version
-        run: |
-          version=$( grep -o "<Version>.*</Version>" Directory.Build.props | grep -o "[0-9]*\.[0-9]*\.[0-9]*")
-          previous_release_tag_version=$(
-            curl -sL https://api.github.com/repos/$GITHUB_REPOSITORY/releases/latest | jq -r ".tag_name"
-          )
-
-          if [ "v$version" == "$previous_release_tag_version" ] && \
-          [ "${{ github.event.inputs.release_type }}" == "Initial Release" ]; then
-            echo "[!] Already released v$version. Please bump version to continue"
-            exit 1
-          fi
-
-          echo "::set-output name=package::$version"
+        uses: bitwarden/gh-actions@master
```

Comment:
This will need a commit once https://github.com/bitwarden/gh-actions/pull/47 has been merged.